workflow/rocoto/gefs_tasks.py

```diff
@@ -264,6 +264,126 @@ def _atmosoceaniceprod(self, component: str):
 
         return task
 
+    def postsnd(self):
+
+        resources = self.get_resource('postsnd')
+
+        deps = []
+        for member in range(0, self.nmem + 1):
+            task = f'{self.run}_atmos_prod_mem{member:03d}_#fhr_label#'
```

Comment:
@AntonMFernando-NOAA Based on the GEFSv12 workflow, I believe the postsnd tasks will depend on the completion of the forecast tasks. 

```suggestion
            task = f'{self.run}_fcst_mem#member#_#seg_dep#'
```